pkg/resourcetopologyexporter/resourcetopologyexporter.go

```diff
@@ -62,7 +63,11 @@ func Execute(hnd resourcemonitor.Handle, nrtupdaterArgs nrtupdater.Args, resourc
 	}
 	go resObs.Run(eventSource.Events(), condChan)
 
-	nodeGetter := nrtupdater.NewNodeGetter(rteArgs.AddNRTOwnerEnable, hnd.K8SCli)
+	nodeGetter := nrtupdater.NewNodeGetter(rteArgs.AddNRTOwnerEnable, hnd.K8SCli, context.TODO())
+	if nil == nodeGetter {
+		klog.V(2).Info("Cannot enable 'add-nrt-owner'.")
```

Comment:
I'd just error out here. We may move this block up to not leak `go resObs.Run()` or anything else

Reply:
ok ... I didn't do it because I was not sure if a fail on that small feature was worthy to fail .